### PR TITLE
docs(error-reporting): improve clarity for opt-out folks

### LINKED ISSUE #3854: Document --no-enable-error-reporting and CI env var
https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md mentions the `--no-enable-error-reporting` but the readme and `lighthouse --help` do not list this flag.

That page also mentions as `CI` env variable. Needs examples and mention of that in  `--help`.

It may also be better to rename the flag `--disable-enable-error-reporting` to be consistent with the other disable flags.

### PR DESCRIPTION
**Pull request by paulirish**
fixes #3854 

* docs clarity around "opt-out" and all that.
* include docs link from --help
* show docs link before providing CLI prompt

### CODE REVIEW

File: lighthouse-cli/cli-flags.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ function getFlags(manualArgv) {
         'Configuration:')
       .describe({
         'enable-error-reporting':
-            'Enables error reporting (prompts once by default, setting this flag will force error reporting to that state).',
+            'Force enables error reporting. --no-enable-error-reporting will force disable. More: https://git.io/vFFTO',
```
Comment (ebidel) [inline lighthouse-cli/cli-flags.js]: "Force enables" sounds like a Jedi move. How about "Enables error reporting, overriding any saved preference...."

Comment (paulirish) [reply lighthouse-cli/cli-flags.js]: > Enables error reporting, overriding any saved preference. --no-enable-error-reporting will do the opposite. More: https://git.io/vFFTO

**Comment by wardpeet** [comment]
should we never prompt again or after x weeks or after each new release?

**Comment by paulirish** [comment]
> should we never prompt again or after x weeks or after each new release?

I'm into the idea of prompting again if we hit that 20sec timeout, but also it may not matter with enough scale.

**Comment by paulirish** [comment]
Should we ship a quick 2.6.1 with this and Brendans new PR?

**Pull request merged**